app/assets/stylesheets/modules/_navigation.scss

```diff
@@ -9,21 +9,21 @@
 
 .nav a { color: #fefeff; }
 
-.navbar-bright {
+.navbar-right {
   background-color:#7f0f14;
   color:#fff;
   margin-bottom: 0;
   border-width: 0;
 }
 
-.navbar-bright a, .navbar-bright .navbar-brand, #masthead h4 {
+.navbar-right a, .navbar-right .navbar-brand, #masthead h4 {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own single line